Configuration/TCA/tx_pxaproductmanager_domain_model_attributevalue.php

```diff
@@ -9,9 +9,17 @@
     $attributes = [];
 }
 
-$types = [];
+$types = [
```

Comment:
BE issue occurs:

Core: Exception handler (WEB): Uncaught TYPO3 Exception: Too few arguments to function TYPO3\CMS\Extbase\Persistence\Repository::__construct(), 0 passed in /var/www/html/public/typo3/sysext/core/Classes/Utility/GeneralUtility.php on line 3431 and exactly 1 expected | ArgumentCountError thrown in file /var/www/html/public/typo3/sysext/extbase/Classes/Persistence/Repository.php in line 68. Requested URL: http://golfstore.t3.localhost/typo3/index.php?route=%%2Frecord%%2Fcommit&token=--AnonymizedToken--&cacheCmd=all